Userland/Libraries/LibJS/Tests/builtins/Array/Array.prototype.splice.js

```diff
@@ -47,3 +47,15 @@ test("basic functionality", () => {
     expect(array).toEqual([]);
     expect(removed).toEqual(["foo", "bar", "baz"]);
 });
+
+test("Invalid lengths", () => {
+    expect(() => {
+        var length = Math.pow(2, 32);
+
+        var obj = {
+            length: length,
+        };
+
+        Array.prototype.splice.call(obj, 0);
+    }).toThrowWithMessage(RangeError, "");
```

Comment:
Please use the specific error message, and only put the code that's expected to throw inside the `expect()` function

Reply:
Whoops... not sure how the test didn't fail.

Reply:
It didn't because the string is treated as a regular expression matched against the actual error message, and an empty string will always match. I'm not aware of us actually using it anywhere, and quite frankly it should just be removed - if you want regex-behaviour, use a regex...